typings/index.d.ts

```diff
@@ -3676,11 +3673,6 @@ export interface IntegrationData {
   type: string;
```

Comment:
This interface (`IntegrationData`) can now be deleted too.

Reply:
removed.